tests/test_modeling_speech_encoder_decoder.py

```diff
@@ -489,6 +505,10 @@ def test_encoder_decoder_model_from_pretrained_configs(self):
     def test_save_and_load_from_pretrained(self):
         pass
 
+    # all published pretrained models are Speech2TextModel != Speech2TextEncoder
+    def test_real_model_save_load_from_pretrained(self):
```

Comment:
Same issue with `Speech2TextModel != Speech2TextEncoder` for the pretrained `facebook/s2t-small-librispeech-asr`, so just disabling it for now.